src/components/store/ProductsCard.vue

```diff
@@ -0,0 +1,77 @@
+<template>
+  <div>
+    <v-card class="mx-auto" max-width="300" color="white" hover
+      ><router-link :to="('store/' + product.name).replace(/\s/g, '')">
+        <v-img
+          :lazy-src="product.thumbnail"
+          :src="product.thumbnail"
+          height="300"
+          width="300"
+          position="right right"
+          ><template v-slot:placeholder>
+            <v-row class="fill-height ma-0" align="center" justify="center">
+              <v-progress-circular
+                indeterminate
+                color="grey lighten-5"
+              ></v-progress-circular>
+            </v-row> </template
+        ></v-img>
+        <v-card-title class="productTittle"> {{ product.name }}</v-card-title>
+        <v-card-subtitle>
+          <p class="productPrice">${{ product.price }}</p>
```

Comment:
I think here, you should use some formatting for the price, you could use the one I used if you want:
```
formatPrice(x) {
            x = Math.round((x + Number.EPSILON) * 100) / 100
            const parts=x.toString().split('.')
            parts[0] = parts[0].replace(/(\d+)(?=\d{3})/g, "$ $1,")
            return parts.join('.')
},
```